### ISSUE TITLE
lidR::readLAS for most las files even lesser 50KB is crashing R

### ISSUE THREAD

**Issue by nidhi23ug** [r-lidar/rlas#59]
Newest version of lidR:: readLas is crashing R, even for the las files that are smaller than 50Kb (files aren't corrupt). I can read them in CloudCompare, python but not in lidR? The error message I got was `An irrecoverable exception occurred. R is aborting now ` `***` caught segfault *** address (nil), cause 'memory not mapped'`

I can send one such file for your testing.

**Comment by Jean-Romain** [comment]
`rlas::read.las("~/Téléchargements/08nd0862.dams__clipped_polygon_2.las")`

Confirmed

**Comment by Jean-Romain** [comment]
Seems to be related to extrabytes

```r
# works
rlas::read.las("~/Téléchargements/08nd0862.dams__clipped_polygon_2.las", select = "")
rlas::read.las("~/Téléchargements/08nd0862.dams__clipped_polygon_2.las", select = "* -0")

# crashes
rlas::read.las("~/Téléchargements/08nd0862.dams__clipped_polygon_2.las", select = "1")
```

**Comment by nidhi23ug** [comment]
great this works. Thank you

**Comment by Jean-Romain** [comment]
I don't where I put the file. Did you send it to me by email. What is your name so I can find your email.

**Comment by dimfalk** [comment]
> [nidhi23ug](https://github.com/nidhi23ug) commented on Jan 21, 2023 via email

Yup, the mail should have the file `08nd0862.dams__clipped_polygon_2.las` attached.

**Comment by Jean-Romain** [comment]
That makes sense... I found it!
